### DISCUSSION TITLE
Problems with template parsing?

### DISCUSSION THREAD

**Discussion by sairson** [projectdiscovery/nuclei#3835] [Q&A]
Same ExecuteOptions, using files byte by templates. ParseTemplateFromReader parsing the template cannot be in the engine, and the store. The Load from the local Load, there is no problem, is this why?

**Comment by sairson** [comment]
![image](https://github.com/projectdiscovery/nuclei/assets/74412075/99196522-70f0-486f-bceb-1eb983c5dfeb) no
![image](https://github.com/projectdiscovery/nuclei/assets/74412075/3b144879-1b41-47aa-b82a-841478a7a192) yes

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
